proto/ory/keto/relation_tuples/v1alpha2/check_service.proto

```diff
@@ -17,7 +20,43 @@ option php_namespace = "Ory\\Keto\\RelationTuples\\v1alpha2";
 // This service is part of the [read-APIs](../concepts/25_api-overview.mdx#read-apis).
 service CheckService {
   // Performs an authorization check.
-  rpc Check(CheckRequest) returns (CheckResponse);
+  rpc Check(CheckRequest) returns (CheckResponse) {
+    option (google.api.http) = {
+      get: "/relation-tuples/check"
+      additional_bindings: {get: "/relation-tuples/check/openapi"}
+      additional_bindings: {
+        post: "/relation-tuples/check"
+        body: "*"
+      }
+      additional_bindings: {
+        post: "/relation-tuples/check/openapi"
+        body: "*"
+      }
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      consumes: "application/x-www-form-urlencoded"
+      consumes: "application/json"
```

Comment:
This says we can submit the payload either as a URL encoded form, or JSON for any of the endpoints defined above right?

Reply:
Yes, because for POST we go through the body, for GET we go through the URL encoded query params.